static/app/utils/metrics.tsx

```diff
@@ -148,6 +133,12 @@ export function useMetricsData({
   );
 }
 
+function getDateTimeParams({start, end, period}: PageFilters['datetime']) {
+  return period
+    ? {statsPeriod: period}
+    : {start: moment(start).toISOString(), end: moment(end).toISOString()};
```

Comment:
We still have to convert these to ISO because they are formatted as locale strings